ocp_resources/task_run.py

```diff
@@ -0,0 +1,61 @@
+# API reference: https://tekton.dev/docs/pipelines/taskruns/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class TaskRun(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        task_ref=None,
+        task_spec=None,
+        params=None,
+        service_account_name=None,
+        taskrun_timeout=None,
+        **kwargs,
+    ):
+        """
+        Args:
+            task_ref or task_spec (str, mandatory): Base task to run taskrun
+                Choose one either task_ref or task_spec
```

Comment:
please fix - the list of args should include each arg by itself; the explanation should include that it is one or the other

Reply:
just to check if i understand correctly:

* I need to define both task_ref and task_spec separately and not in one argument
* so do i need to add that either one needs to be picked in both places ( i.e under task_ref and task_sepc)

task_ref  (str, mandatory): Base task to run taskrun. Optional, if task_spec is provided.
task_spec  (str, mandatory): Base task to run taskrun. Optional, if task_ref is provided.

Reply:
@rnetser ^^ Hey you mean these changes

Reply:
```
task_ref (str): Base task to run taskrun. Mandatory if task_spec is not provided.
task_spec (str): Base task to run taskrun. Mandatory if task_ref is not provided.
```
if it is one or the other, `task_ref` cannot be optional `task_spec` is provided (there's an exception if both are provided)